unlock-app/src/components/interface/verification/Key.tsx

```diff
@@ -205,14 +207,17 @@ export const ValidKey = ({
         return
       }
       await siweLogin()
+
+      // get lockAddress for lock that use a past version of Unlock contract
+      const lockAddress = lock?.address || lock?.unlockContractAddress
```

Comment:
That is not the lock address. Unlock contract is different from lock contract!!!